kibana-reports/public/components/main/report_definition_details/report_definition_details.tsx

```diff
@@ -392,6 +427,59 @@ export function ReportDefinitionDetails(props) {
       <ScheduledDefinitionStatus />
     );
 
+  const triggerSection =
+    reportDefinitionDetails.triggerType === 'On demand' ? (
```

Comment:
When same string is used multiple time then may be use constant?

Reply:
Sure, will add